frontends/mit-open/package.json

```diff
@@ -42,6 +43,7 @@
     "html-webpack-plugin": "^5.6.0",
     "mini-css-extract-plugin": "^2.6.1",
     "ol-test-utilities": "0.0.0",
+    "react-refresh": "^0.14.2",
```

Comment:
This is a peer-dependency of another package we use, `@pmmmwh/react-refresh-webpack-plugin`. Previously we were not installing the peer dep explicitly (it was added by react-scripts, I believe). Now that the `github-pages` workspace is removed, we needed to explicitly install it.